### DISCUSSION TITLE
farming for other crypto, with CHIA rig ?

### DISCUSSION THREAD

**Discussion by HappyHarris** [Chia-Network/chia-blockchain#6216] [General]
Am I able to use my Chia rig, to farm for other coins/crpyto ?

**Comment by tiny0little** [comment]
yes, you can definitely do that. for example mine Monero, as it uses CPU, and mine a coin which uses GPU. you'll end up mining 3 coins at the same time on one PC. just make sure you have enough resources for that, for example in case of CHIA you need to make sure no WARNING like this in the logs

```WARNING  Looking up qualities on <plot> took: 5.076157808303833. This should be below 5 seconds to minimize risk of losing rewards```

**Reply by joelrb** [reply]
How to make it below 5 seconds? It seems automatically done.

**Reply by tiny0little** [reply]
it is related to the performance of your hard drives {and possibly CPU}. often this WARNING appears when one plots and farms on the same hard drive. it is a requirement to have plot's quality validation done under 5 sec in order not to risk loosing possible reward

**Comment by HappyHarris** [comment]
Thanks Nickolay, very appreciated 

David
